### ISSUE TITLE
Subcutaneous ossification

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7551]
**Preferred term label:**
Subcutaneous ossification

**Synonyms**

**Definition (free text, please give PubMed ID)**
Formation of abnormal, extraskeletal bony tissue in the soft tissue beneath the skin. Subcutaneous ossifications may be observed by radiography or by palpation.
See the Figure in PMID:29059381

**Parent term (use hpo.jax.org/app)**
Ectopic ossification HP:0011986

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
103580 
PMID: 2109828  2/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
done
